### DISCUSSION TITLE
Black list (?)

### DISCUSSION THREAD

**Discussion by ghost** [Koenkk/zigbee2mqtt#12880] [General]
How do you explain otherwise that:
1 button Samsung
1 switch Vimar
1 sensor air quality Frient

that worked for months, after update of zigbee2mqtt version or after un-pairing , now is not possible make pairing?

I have 2 button Samsung, 1 works fine, the other not pair
4 switch Vimar, 3 works fine, 1 other not pair
2 sensor air quality Frient, 1 works fine, the other not pair

When the device is new no problem, but if it's on "old" device and one time the pairing fails, then the device is lost, like there's a black list of devices faulty ....

**Comment by ghost** [comment]
It's obvious that there is no a black list, but there must be a reason ...
@Koenkk do you think do us the honour to give an plausible explanation?

**Reply by Koenkk** [reply]
Please check https://www.zigbee2mqtt.io/guide/faq/#no-logging-is-shown-at-all

**Comment by ghost** [comment]
@Koenkk

Before ..... Please check https://www.zigbee2mqtt.io/guide/faq/#no-logging-is-shown-at-all

I I tried for one time to pairing a Vimar switch and like the other times the switch seems okay but in fact I have this situation:

Zigbee2MQTT:info  2022-06-20 21:31:09: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"definition":{"description":"2-way switch IoT connected mechanism","exposes":[{"features":[{"access":7,"description":"On/off state of the switch","name":"state","property":"state","type":"binary","value_off":"OFF","value_on":"ON","value_toggle":"TOGGLE"}],"type":"switch"},{"access":1,"description":"Link quality (signal strength)","name":"linkquality","property":"linkquality","type":"numeric","unit":"lqi","value_max":255,"value_min":0}],"model":"14592.0","options":[],"supports_ota":false,"vendor":"Vimar"},"friendly_name":"0xf4ce36660fdb39f1","ieee_address":"0xf4ce36660fdb39f1","status":"successful","supported":true},"type":"device_interview"}'

Zigbee2MQTT:info  2022-06-20 21:31:09: Successfully configured '0xf4ce36660fdb39f1'

Zigbee2MQTT:info  2022-06-20 21:37:20: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'get' 'state' to '0xf4ce36660fdb39f1' failed: 'Error: Read 0xf4ce36660fdb39f1/10 genOnOff([\"onOff\"], {\"sendWhen\":\"immediate\",\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":true,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null,\"writeUndiv\":false}) failed (no response received)'","meta":{"friendly_name":"0xf4ce36660fdb39f1"},"type":"zigbee_publish_error"}'

Zigbee2MQTT:error 2022-06-20 21:37:20: Publish 'get' 'state' to '0xf4ce36660fdb39f1' failed: 'Error: Read 0xf4ce36660fdb39f1/10 genOnOff(["onOff"], {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received)'

and the switch don't work ...

![Screenshot_20220620_213212](https://user-images.githubusercontent.com/105095607/174670778-67fafad2-b116-4880-b63c-6afdb04a196d.png)

**Reply by Koenkk** [reply]
- Can you provide the data/database.db entry of this device?
- Does only the refresh fail (read) or also the changing of state?

**Reply by ghost** [reply]
Well, I did some test with another switch Vimar.
I rebooted my Raspberry Pi4 and my home automation (node-red - zigbee2mqtt) with all the devices (sensor temp - sensor door - sensor motion, plug smart, ecc. ecc) works very well.
This switch in the past worked fine like the other, but after update zigbee2mqtt to version 1.25.0 stopping work
Today this is the story ... (to the switch I attacked one physical switch and one lamp)
The pairing seems ok and on video I see the 5 green notice (configuring, pairing ok, ecc. ecc) but this is the situation:

![img1](https://user-images.githubusercontent.com/105095607/175030880-068b0c4f-e4bc-4aba-ae1a-2e05a4071ebf.png)

![img2](https://user-images.githubusercontent.com/105095607/175030915-32b2ebcc-3f74-45bc-b07f-5df31bf4b635.png)

the LQI is N/A and the state is ?
Strangely led of the switch blinks white how like it was always in pairing mode ...
Then I push the physical switch four time (on-off on-off) and this is the new situation:

![img3](https://user-images.githubusercontent.com/105095607/175032435-85a3c77b-059c-48d1-a297-df4f4b1bccfe.png)

![img4](https://user-images.githubusercontent.com/105095607/175032459-65696917-c026-43d6-b347-4d3e4af74649.png)

Led of the switch blinks white again and then I push Disable join in ui Z2M and I change the name in LuceSalotto (the led stop blink).

It seems ok, but 
if I push the physical switch the lamp turn on-off but the state don't change and I have an error,
If I click on video ON or OFF the lamp turn on-off but the state don't change and I have an error,
and the state don't arrive to node-red of course.

This is the db:
{"id":28,"type":"Router","ieeeAddr":"0xf4ce36cb2e96b9da","nwkAddr":1927,"manufId":4660,"manufName":"Vimar","powerSource":"Mains (single phase)","modelId":"On_Off_Switch_v1.0","epList":[10,242],"endpoints":{"10":{"profId":260,"epId":10,"devId":0,"inClusterList":[0,6,7,3,5,4],"outClusterList":[6,3],"clusters":{"genBasic":{"attributes":{"modelId":"On_Off_Switch_v1.0","manufacturerName":"Vimar","powerSource":1,"zclVersion":2,"appVersion":1,"stackVersion":10,"hwVersion":11,"dateCode":"20200501","swBuildId":""}},"genOnOff":{"attributes":{"onOff":0}}},"binds":[{"cluster":6,"type":"endpoint","deviceIeeeAddress":"0x00212effff085bf5","endpointID":1}],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":1,"stackVersion":10,"hwVersion":11,"dateCode":"20200501","swBuildId":"","zclVersion":2,"interviewCompleted":true,"meta":{"configured":3},"lastSeen":1655894618698,"defaultSendRequestWhen":"immediate"}

This is the pairing:
Zigbee2MQTT:info  2022-06-22 12:40:47: Device '0xf4ce36cb2e96b9da' joined
Zigbee2MQTT:info  2022-06-22 12:40:47: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0xf4ce36cb2e96b9da","ieee_address":"0xf4ce36cb2e96b9da"},"type":"device_joined"}'
Zigbee2MQTT:info  2022-06-22 12:40:47: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":{"friendly_name":"0xf4ce36cb2e96b9da"},"type":"device_connected"}'
Zigbee2MQTT:info  2022-06-22 12:40:47: Starting interview of '0xf4ce36cb2e96b9da'
Zigbee2MQTT:info  2022-06-22 12:40:47: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0xf4ce36cb2e96b9da","ieee_address":"0xf4ce36cb2e96b9da","status":"started"},"type":"device_interview"}'
Zigbee2MQTT:info  2022-06-22 12:40:47: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_started","meta":{"friendly_name":"0xf4ce36cb2e96b9da"},"type":"pairing"}'
Zigbee2MQTT:info  2022-06-22 12:40:48: Successfully interviewed '0xf4ce36cb2e96b9da', device has successfully been paired
Zigbee2MQTT:info  2022-06-22 12:40:48: Device '0xf4ce36cb2e96b9da' is supported, identified as: Vimar 2-way switch IoT connected mechanism (14592.0)

Zigbee2MQTT:info  2022-06-22 12:40:48: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"definition":{"description":"2-way switch IoT connected mechanism","exposes":[{"features":[{"access":7,"description":"On/off state of the switch","name":"state","property":"state","type":"binary","value_off":"OFF","value_on":"ON","value_toggle":"TOGGLE"}],"type":"switch"},{"access":1,"description":"Link quality (signal strength)","name":"linkquality","property":"linkquality","type":"numeric","unit":"lqi","value_max":255,"value_min":0}],"model":"14592.0","options":[],"supports_ota":false,"vendor":"Vimar"},"friendly_name":"0xf4ce36cb2e96b9da","ieee_address":"0xf4ce36cb2e96b9da","status":"successful","supported":true},"type":"device_interview"}'
Zigbee2MQTT:info  2022-06-22 12:40:48: Configuring '0xf4ce36cb2e96b9da'
Zigbee2MQTT:info  2022-06-22 12:40:48: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_successful","meta":{"description":"2-way switch IoT connected mechanism","friendly_name":"0xf4ce36cb2e96b9da","model":"14592.0","supported":true,"vendor":"Vimar"},"type":"pairing"}'
Zigbee2MQTT:info  2022-06-22 12:40:48: Successfully configured '0xf4ce36cb2e96b9da'

This is the errror:
Zigbee2MQTT:error 2022-06-22 14:21:09: Publish 'set' 'state' to 'LuceSalotto' failed: 'Error: Command 0xf4ce36cb2e96b9da/10 genOnOff.off({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received)'

I hope this information helps you!

**Reply by Koenkk** [reply]
It looks to me that somehow the switch disconnects from the network. Can you provide the herdsman debug log of, before every capture first force remove the device via the z2m frontend and restart z2m:
- Pre z2m 1.25.0 (where the device worked without issues)
  - Pair device (goes OK)
  - Control device (goes OK)
- z2m 1.25.2
  - Pair device (goes OK)
  - Control device (fails)

See https://www.zigbee2mqtt.io/guide/usage/debug.html on how to enable the herdsman debug logging. Note that this is only logged to STDOUT and not to log files.

**Reply by ghost** [reply]
. remove the device via the z2m frontend 
. restart z2m with herdsman debug logging

PAIRING:
![img01](https://user-images.githubusercontent.com/105095607/175081639-fb945c77-5e88-4d89-89c0-3765985412e0.png)

but LQI = N/A and as written before after push the physical switch four time (on-off on-off) the situation is:

![img02](https://user-images.githubusercontent.com/105095607/175082341-78c96eff-aea9-492a-8115-602f96834687.png)

![xxxxx](https://user-images.githubusercontent.com/105095607/175082759-334c7aa5-ac1d-41d0-9555-9cf69abd2374.png)

DEBUG:
zigbee-herdsman:deconz:driver send read device state from queue. seqNr: 176 +10s
  zigbee-herdsman:deconz:frameParser device state: 10100010 +10s
  zigbee-herdsman:deconz:driver networkstate: 2 apsDataConfirm: 0 apsDataIndication: 0 configChanged: 0 apsRequestFreeSlots: 1 +3ms
  zigbee-herdsman:deconz:adapter BIND FAILED - addr: 0xa79a waiting for response TIMEOUT +20s
  zigbee-herdsman:controller:endpoint Bind 0xf4ce36cb2e96b9da/10 genOnOff from '0x00212effff085bf5/1' failed (waiting for response TIMEOUT) +3m
Zigbee2MQTT:error 2022-06-22 17:59:09: Failed to configure '0xf4ce36cb2e96b9da', attempt 1 (Error: Bind 0xf4ce36cb2e96b9da/10 genOnOff from '0x00212effff085bf5/1' failed (waiting for response TIMEOUT)
    at DeconzAdapter.bind (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/adapter/deconz/adapter/deconzAdapter.ts:765:19)
    at Endpoint.bind (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/controller/model/endpoint.ts:567:13)
    at Object.bind (/opt/zigbee2mqtt/node_modules/zigbee-herdsman-converters/lib/reporting.js:40:9)
    at Object.configure (/opt/zigbee2mqtt/node_modules/zigbee-herdsman-converters/devices/vimar.js:17:13)
    at Configure.configure (/opt/zigbee2mqtt/lib/extension/configure.ts:115:13))
  zigbee-herdsman:deconz:driver send read device state from queue. seqNr: 177 +10s
  zigbee-herdsman:deconz:frameParser device state: 10100010 +10s
  zigbee-herdsman:deconz:driver networkstate: 2 apsDataConfirm: 0 apsDataIndication: 0 configChanged: 0 apsRequestFreeSlots: 1 +2ms
  zigbee-herdsman:deconz:frameParser DEVICE_STATE changed: 10101010 +7s
  zigbee-herdsman:deconz:driver networkstate: 2 apsDataConfirm: 0 apsDataIndication: 1 configChanged: 0 apsRequestFreeSlots: 1 +7s
  zigbee-herdsman:deconz:driver query aps data indication +2ms
  zigbee-herdsman:deconz:driver DATA_INDICATION - sending read data request - SeqNr. 178 +4ms
  zigbee-herdsman:deconz:frameParser DATA_INDICATION RESPONSE - seqNr. 178 srcAddr: 0x31f7 destAddr: 0xfffd profile id: 0x0 cluster id: 0x13 lqi: 255 +12ms
  zigbee-herdsman:deconz:frameParser response payload: 134,247,49,240,105,8,1,0,151,109,40,128 +0ms
  zigbee-herdsman:controller:log Device '0x286d9700010869f0' joined +28s
  zigbee-herdsman:deconz:driver networkstate: 2 apsDataConfirm: 0 apsDataIndication: 0 configChanged: 0 apsRequestFreeSlots: 1 +7ms

WHEN I PUSH IN UI Z2M state ON DEBUG IS:
zigbee-herdsman:controller:endpoint Command 0xf4ce36cb2e96b9da/10 genOnOff.on({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) +31s
  zigbee-herdsman:deconz:driver DATA_REQUEST - destAddr: 0xe35 EP:10 SeqNr. 34 request id: 55 +4s
  zigbee-herdsman:deconz:frameParser DATA_REQUEST RESPONSE - request id: 55 status: 0 +4s
  zigbee-herdsman:deconz:driver networkstate: 2 apsDataConfirm: 0 apsDataIndication: 0 configChanged: 0 apsRequestFreeSlots: 1 +11ms
  zigbee-herdsman:deconz:driver send read device state from queue. seqNr: 35 +6s
  zigbee-herdsman:deconz:frameParser device state: 10100010 +6s
  zigbee-herdsman:deconz:driver networkstate: 2 apsDataConfirm: 0 apsDataIndication: 0 configChanged: 0 apsRequestFreeSlots: 1 +4ms
  zigbee-herdsman:deconz:frameParser DEVICE_STATE changed: 10100110 +4s
  zigbee-herdsman:deconz:driver networkstate: 2 apsDataConfirm: 1 apsDataIndication: 0 configChanged: 0 apsRequestFreeSlots: 1 +4s
  zigbee-herdsman:deconz:driver query aps data confirm +3ms
  zigbee-herdsman:deconz:driver DATA_CONFIRM - sending data state request - SeqNr. 36 +5ms
  zigbee-herdsman:deconz:frameParser DATA_CONFIRM RESPONSE - destAddr: 0xe35 request id: 55 confirm status: 0 +9ms
  zigbee-herdsman:deconz:driver networkstate: 2 apsDataConfirm: 0 apsDataIndication: 0 configChanged: 0 apsRequestFreeSlots: 1 +2ms
  zigbee-herdsman:deconz:adapter sendZclFrameToEndpoint - message send with transSeq Nr.: 18 +41s
  zigbee-herdsman:deconz:adapter false, false, false +0ms
  zigbee-herdsman:deconz:adapter resolve request +0ms
  zigbee-herdsman:deconz:frameParser DEVICE_STATE changed: 10101010 +2s
  zigbee-herdsman:deconz:driver networkstate: 2 apsDataConfirm: 0 apsDataIndication: 1 configChanged: 0 apsRequestFreeSlots: 1 +2s
  zigbee-herdsman:deconz:driver query aps data indication +1ms
  zigbee-herdsman:deconz:driver DATA_INDICATION - sending read data request - SeqNr. 37 +5ms
  zigbee-herdsman:deconz:frameParser DATA_INDICATION RESPONSE - seqNr. 37 srcAddr: 0xf335 destAddr: 0x0 profile id: 0x104 cluster id: 0x6 lqi: 248 +10ms
  zigbee-herdsman:deconz:frameParser response payload: 8,91,10,0,0,16,0 +0ms
  zigbee-herdsman:controller:log Received 'zcl' data '{"frame":{"Header":{"frameControl":{"frameType":0,"manufacturerSpecific":false,"direction":1,"disableDefaultResponse":false,"reservedBits":0},"transactionSequenceNumber":91,"manufacturerCode":null,"commandIdentifier":10},"Payload":[{"attrId":0,"dataType":16,"attrData":0}],"Command":{"ID":10,"name":"report","parameters":[{"name":"attrId","type":33},{"name":"dataType","type":32},{"name":"attrData","type":1000}]}},"address":62261,"endpoint":10,"linkquality":248,"groupID":null,"wasBroadcast":false,"destinationEndpoint":1}' +43s

**Reply by ghost** [reply]
I tried with a third switch and also this switch behaved exactly like the one before .....
Pairing ok - LQI 255 - but state ?
and the switch don't work of course ....
How is it possible that everyone behave the same?

Now I write a bullshit .... it's like, the pairing after the first error, did not distinguish them any more, but "see" this switchs always the same and not different devices ...

**Reply by ghost** [reply]
And I'll tell you also this ....
This morning I tried to pairing a Vimar switch again ....
The led of switch blink white and I wait the pairing ....
The led now is fix blue but I didn't see any green notice and I don't see the switch in list device ui Z2M.
For me this switch is connected not to the router but with another switch vimar or is linked like another switch already present
In fact another switch Vimar (LuceCucina) that worked well now don't work and neither the other ...

This is the other switch error:
Zigbee2MQTT:error 2022-06-23 10:34:10: Publish 'set' 'state' to 'LuceCucina' failed: 'Error: Command 0xf4ce367bd01a7061/10 genOnOff.off({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (no response received)'

This morning I pairing a siren without problem, pairing very fast, the siren works fine!
But the Vimar switch ....... I'll go crazy ....

**Reply by ghost** [reply]
This morning I tried to pairing the switch (LuceCucina) that yesterday has disconnected after the failed attempt to pair another switch;
well this is the situation ....

![Screensh080347](https://user-images.githubusercontent.com/105095607/175475196-8bb8e402-f065-4f5a-878c-cdac6e4c7ed7.png)

I don't know what to do...

@Koenkk tell me  you have a solution

**Reply by ghost** [reply]
I tried to pairing another switch and I have ...

![new](https://user-images.githubusercontent.com/105095607/175545760-92e9ce05-8cb7-46fd-b95e-dd0dc7e55c4c.png)

so I compared the db of this switch with another Vimar that work fine and I have ...

**Bedroom**
{"id":22,"type":"Router","ieeeAddr":"0xf4ce36eb94e0a9a0","nwkAddr":17046,"manufId":4660,"manufName":"Vimar","powerSource":"Mains (single phase)","modelId":"On_Off_Switch_v1.0","epList":[10,242],"endpoints":{"10":{"profId":260,"epId":10,"devId":0,"inClusterList":[0,6,7,3,5,4],"outClusterList":[6,3],"clusters":{"genBasic":{"attributes":{"modelId":"On_Off_Switch_v1.0","manufacturerName":"Vimar","powerSource":1,"zclVersion":2,"appVersion":

**1,"stackVersion":10,"hwVersion":11,"dateCode":"20200501","swBuildId":""}},"genOnOff":{"attributes":{"onOff":0}}},"binds":[{"cluster":**

6,"type":"endpoint","deviceIeeeAddress":"0x00212effff085bf5","endpointID":1}],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":1,"stackVersion":10,"hwVersion":11,"dateCode":"20200501","swBuildId":"","zclVersion":2,"interviewCompleted":true,"meta":{"configured":3},"lastSeen":1656075503436,"defaultSendRequestWhen":"immediate"}


**New**
{"id":20,"type":"Router","ieeeAddr":"0xf4ce36cb2e96b9da","nwkAddr":5530,"manufId":4660,"manufName":"Vimar","powerSource":"Mains (single phase)","modelId":"On_Off_Switch_v1.0","epList":[10,242],"endpoints":{"10":{"profId":260,"epId":10,"devId":0,"inClusterList":[0,6,7,3,5,4],"outClusterList":[6,3],"clusters":{"genBasic":{"attributes":{"modelId":"On_Off_Switch_v1.0","manufacturerName":"Vimar","powerSource":1,"zclVersion":2,"appVersion":

**1,"stackVersion":10,"hwVersion":11,"dateCode":"20200501","swBuildId":""}}},"binds":[{"cluster":**

6,"type":"endpoint","deviceIeeeAddress":"0x00212effff085bf5","endpointID":1}],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":1,"stackVersion":10,"hwVersion":11,"dateCode":"20200501","swBuildId":"","zclVersion":2,"interviewCompleted":true,"meta":{"configured":3},"lastSeen":1656075669935,"defaultSendRequestWhen":"immediate"}

You can see the difference; is this the problem?
Can I manually update the database?

**Comment by ghost** [comment]
@Koenkk Hi, I tried to write the largest number of information, can you help me?

**Comment by Koenkk** [comment]
I'm not sure what goes wrong, I would need a sniff (which requires separate hardware) to debug this further, see https://www.zigbee2mqtt.io/advanced/zigbee/04_sniff_zigbee_traffic.html#with-cc2531 how to make one

**Reply by ghost** [reply]
Too difficult for my chances that link ....
This thing is really absurd, until a few months ago all my Vimar work fine. This morning only two work fine, this afternoon  after few tries to pair a switch (always Vimar of course), are always down ....
